src/main/java/ch/njol/skript/util/PotionEffectUtils.java

```diff
@@ -99,32 +95,30 @@ public static PotionEffectType parseByEffectType(PotionEffectType t) {
 		return null;
 	}
 	
-	@SuppressWarnings("null")
-	public static String toString(final PotionEffectType t) {
+	public static String toString(PotionEffectType t) {
 		return names[t.getId()];
 	}
 	
 	// REMIND flags?
-	@SuppressWarnings("null")
-	public static String toString(final PotionEffectType t, final int flags) {
+	public static String toString(PotionEffectType t, int flags) {
 		return names[t.getId()];
 	}
-	
+
 	public static String toString(PotionEffect potionEffect) {
 		StringBuilder builder = new StringBuilder();
 		if (potionEffect.isAmbient())
 			builder.append("ambient ");
 		builder.append("potion effect of ");
 		builder.append(toString(potionEffect.getType()));
-		
 		builder.append(" of tier ").append(potionEffect.getAmplifier() + 1);
-		
 		if (!potionEffect.hasParticles())
 			builder.append(" without particles");
-		builder.append(" for ").append(Timespan.fromTicks_i(potionEffect.getDuration()));
+		builder.append(" for ").append(potionEffect.getDuration() == PotionEffect.INFINITE_DURATION ? "infinity" : Timespan.fromTicks_i(Math.abs(potionEffect.getDuration())));
```

Comment:
```suggestion
		builder.append(" for ").append(potionEffect.getDuration() == -1 ? "infinity" : Timespan.fromTicks_i(Math.abs(potionEffect.getDuration())));
```
```suggestion
		builder.append(" for ").append(potionEffect.getDuration() == -1 ? "forever" : Timespan.fromTicks_i(Math.abs(potionEffect.getDuration())));
```
Do you think this sounds better? I'll let you decide :)

Also changed the constant to -1 since it won't exist on other versions (my bad with the previous comment)

Reply:
for forever is not better